enzyme/Enzyme/EnzymeLogic.cpp

```diff
@@ -993,20 +993,27 @@ getDefaultFunctionTypeForAugmentation(FunctionType *called, bool returnUsed,
 
 //! assuming not top level
 std::pair<SmallVector<Type *, 4>, SmallVector<Type *, 4>>
-getDefaultFunctionTypeForGradient(FunctionType *called, DIFFE_TYPE retType) {
+getDefaultFunctionTypeForGradient(FunctionType *called, DIFFE_TYPE retType,
+                                  const ArrayRef<DIFFE_TYPE> &tys) {
```

Comment:
```suggestion
                                 ArrayRef<DIFFE_TYPE> tys) {
```

`ArrayRef` is already a reference and immutable.